ports/esp32/machine_can.c

```diff
@@ -102,50 +101,47 @@ STATIC mp_obj_t machine_hw_can_init_helper(machine_can_obj_t *self, size_t n_arg
 STATIC void machine_hw_can_print(const mp_print_t *print, mp_obj_t self_in, mp_print_kind_t kind);
 
 // singleton CAN device object
-machine_can_config_t can_config = {.general = CAN_GENERAL_CONFIG_DEFAULT(2, 4, 0),
-                                   .filter = CAN_FILTER_CONFIG_ACCEPT_ALL(),
-                                   .timing = CAN_TIMING_CONFIG_25KBITS(),
-                                   .initialized = false};
+machine_can_config_t can_config = { .general = CAN_GENERAL_CONFIG_DEFAULT(2, 4, 0),
+                                    .filter = CAN_FILTER_CONFIG_ACCEPT_ALL(),
+                                    .timing = CAN_TIMING_CONFIG_25KBITS(),
+                                    .initialized = false };
 
-STATIC machine_can_obj_t machine_can_obj = {{&machine_can_type}, .config = &can_config};
+STATIC machine_can_obj_t machine_can_obj = { {&machine_can_type}, .config = &can_config };
 
-// INTERNAL FUNCTION Return status information
 STATIC can_status_info_t _machine_hw_can_get_status(){
     can_status_info_t status;
     uint32_t err_code = can_get_status_info(&status);
-    if (err_code != ESP_OK)
-    {
+    if (err_code != ESP_OK){
         mp_raise_OSError(-err_code);
     }
     return status;
 }
 
-//INTERNAL FUNCTION Populates the filter register according to inputs
 STATIC void _machine_hw_can_set_filter(machine_can_obj_t *self, uint32_t addr, uint32_t mask, uint8_t bank, bool rtr){
     //Check if bank is allowed
-    if (bank < 0 && bank > ((self->extframe && self->config->filter.single_filter) ? 0 : 1)){
+    if ( bank < 0 && bank > ( (self->extframe && self->config->filter.single_filter) ? 0 : 1 ) ){
```

Comment:
Space around parens not required here. The changes I was suggesting only applied to braces. Please undo these changes.